### PR TITLE
ENH: added api_rst_coverage.py

### LINKED ISSUE #8163: DOC: {Series,DataFrame}.dot() not listed under API Reference
Any reason `{Series,DataFrame}.dot()` are not listed under http://pandas.pydata.org/pandas-docs/stable/api.html?

They are documented:
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.dot.html
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.dot.html

### PR DESCRIPTION
**Pull request by seth-p**
Closes https://github.com/pydata/pandas/issues/8163

### CODE REVIEW

**Comment by jreback** [comment]
@jorisvandenbossche what do you think on this?

**Comment by jreback** [comment]
@seth-p can you add a note in using this in ``contributing.rst``? thxs.

**Comment by seth-p** [comment]
Added a note about ``scripts/api_rst_coverage.py`` to ``contributing.rst``. Let me know if it looks ok.

**Comment by jreback** [comment]
@jorisvandenbossche ?

**Comment by jreback** [comment]
@jorisvandenbossche look ok?

**Comment by jorisvandenbossche** [comment]
I have some ideas to improve it, but won't be able to do that right now, so let's merge this as a good start!

Merged via https://github.com/pydata/pandas/commit/83d4dfd3e3ba20693605ab3e3129e2b7ed42b785

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -303,6 +303,13 @@ Some other important things to know about the docs:
   output saved) during the doc build. This way, they will always be up to date,
   but it makes the doc building a bit more complex.
 
+- The utility script ``scripts/api_rst_coverage.py`` can be used to compare
+  the list of methods documented in ``doc/source/api.rst`` (which is used to generate
+  the `API Reference <http://pandas.pydata.org/pandas-docs/version/0.16.0/api.html>`_ page)
+  and the actual methods of the classes ``Series``, ``DataFrame``, ``Panel``, and ``Panel4D``.
+  It will identify methods documented in in ``doc/source/api.rst`` that are not actually
+  class methods, and class methods that are not documented in ``doc/source/api.rst``.
```
Comment (jreback) [inline doc/source/contributing.rst]: didn't nice this but should point to latest

Comment (jorisvandenbossche) [inline doc/source/contributing.rst]: yep, I changed that in what I actually committed

**Pull request closed without merging**